x-pack/plugins/triggers_actions_ui/common/experimental_features.ts

```diff
@@ -13,7 +13,7 @@ export type ExperimentalFeatures = typeof allowedExperimentalValues;
  */
 export const allowedExperimentalValues = Object.freeze({
   rulesListDatagrid: true,
-  internalAlertsTable: false,
+  internalAlertsTable: true,
```

Comment:
Should we revert this?

Reply:
oh yes

Reply:
f3f2522df1a63c9c8e5a9b809bad742da380f743